pkgs/applications/audio/gtkcord4/default.nix

```
@@ -36,8 +38,18 @@ buildGoModule rec {
     graphene
     gtk4
     pango
+  ] ++ lib.optionals withLibadwaita [
+    libadwaita
   ];
 
+  tags = lib.optionals withLibadwaita [ "libadwaita" ];
+
+  postInstall = ''
+    install -D -m 444 -t ${placeholder "out"}/share/applications .nix/com.github.diamondburned.gtkcord4.desktop
+    install -D -m 444 internal/icons/svg/logo.svg ${placeholder "out"}/share/icons/hicolor/scalable/apps/gtkcord4.svg
+    install -D -m 444 internal/icons/png/logo.png ${placeholder "out"}/share/icons/hicolor/256x256/apps/gtkcord4.png
```

while this does work, I think `$out` reads better:
```suggestion
    install -D -m 444 -t $out/share/applications .nix/com.github.diamondburned.gtkcord4.desktop
    install -D -m 444 internal/icons/svg/logo.svg $out/share/icons/hicolor/scalable/apps/gtkcord4.svg
    install -D -m 444 internal/icons/png/logo.png $out/share/icons/hicolor/256x256/apps/gtkcord4.png
```